tests/run_gen.py

```diff
@@ -56,9 +56,6 @@ def run_gen(fn):
     return run_generator(
         program_data,
         env_data,
-        KEYWORD_TO_ATOM["q"][0],
-        KEYWORD_TO_ATOM["a"][0],
-        native_opcode_names_by_opcode,
```

Comment:
again, I think we should hold off on this API change, and bump the version number at the same time